osx/main.py

```diff
@@ -0,0 +1,104 @@
+import json
+import urllib2
+from os.path import expanduser
+
+ENVZ = 'envz'
+SVCZ = 'svcz'
+NOOP = 'noop'
+required_keys = [ENVZ, SVCZ]
+super_secret_proxy_tmpl = "http://status.bign8.info/proxy?url={}"
```

Comment:
It caches results for 1s right now to not exhert a large load on any particular server (it is configurable if you would like to spin up your own server).

https://github.com/bign8/status.bign8.info/blob/master/main.go#L38